docs/run-local.md

```diff
@@ -40,6 +40,13 @@ You may wish to run the dashboard. Instructions are [here](run-dashboard.md).
 
 You may wish to run Rancher rather than the dashboard. Rancher provide a step by step walkthrough to install, which is linked [here](rancher-setup.md)
 
+## Add Dependencies
+
+In order to run locally, a dependancy to the ACR must be added to Helm. This can be done using the below command
+```
+helm repo add gpitfuturesdevacr https://gpitfuturesdevacr.azurecr.io/helm/v1/repo --username gpitfuturesdevacr --password $(gpitfuturesdevacr-pass)
```

Comment:
How do I know where to get `$(gpitfuturesdevacr-pass)` information?

Reply:
You would already have it if starting from scratch (it's needed for the Private registry a couple of steps above)

Reply:
We should probably add clearer instructions that it's the same password as used for the local docker command

Reply:
In that case doesn't this section belong in the Private registry section?